sqlactivitylog/sqlactivitylog.py

```diff
@@ -185,7 +185,6 @@ async def togglelock(self, ctx):
         await ctx.send(inline('Locked is now {}'.format(self.lock)))
 
     @commands.group()
-    @commands.guild_only()
     @checks.mod_or_permissions(manage_guild=True)
```

Comment:
Why is guild_only removed here?

Reply:
because it works fine out of a guild.  all of them except for user do actually